embedchain/config/QueryConfig.py

```diff
@@ -13,25 +13,73 @@
   Helpful Answer:
 """
 
+DEFAULT_PROMPT_WITH_HISTORY = """
+  Use the following pieces of context to answer the query at the end.
+  If you don't know the answer, just say that you don't know, don't try to make up an answer.
+  I will provide you with our conversation history. 
+
+  $context
+
+  History: $history
+
+  Query: $query
+
+  Helpful Answer:
+"""
+
 DEFAULT_PROMPT_TEMPLATE = Template(DEFAULT_PROMPT)
+DEFAULT_PROMPT_WITH_HISTORY_TEMPLATE = Template(DEFAULT_PROMPT_WITH_HISTORY)
 query_re = re.compile(r"\$\{*query\}*")
 context_re = re.compile(r"\$\{*context\}*")
+history_re = re.compile(r"\$\{*context\}*")
```

Comment:
Typo: this should be `history` instead of `context`

Reply:
good catch